<launch>
    <arg name="conf_location" default="~/.config/niryo"/>

    <node name="niryo_robot_mqtt_client" pkg="niryo_robot_iot" type="mqtt_client_node.py" output="screen">
        <rosparam file="$(find niryo_robot_iot)/config/aws_const.yaml"/>
        <rosparam file="$(find niryo_robot_iot)/config/iot_params.yaml"/>
        <param name="conf_location" value="$(arg conf_location)"/>
        <param name="s3_credentials_location" value="$(arg conf_location)/credentials"/>
    </node>
</launch>
